src/Host.h

```diff
@@ -162,48 +155,28 @@ class Host : public QObject
     int mBorderLeftWidth;
     int mBorderRightWidth;
     int mBorderTopHeight;
-    QString mBufferIncomingData;
-    bool mCodeCompletion;
     QFont mCommandLineFont;
     QString mCommandSeparator;
-    bool mDisableAutoCompletion;
     QFont mDisplayFont;
     bool mEnableGMCP;
     bool mEnableMSDP;
-    int mEncoding;
     QTextStream mErrorLogStream;
-    QFile mErrorLogFile;
     QMap<QString, QList<TScript*>> mEventHandlerMap;
-    QMap<QString, TEvent*> mEventMap;
     bool mFORCE_GA_OFF;
     bool mFORCE_NO_COMPRESSION;
     bool mFORCE_SAVE_ON_EXIT;
-    int mHostID;
-    QString mHostName;
     bool mInsertedMissingLF;
-    bool mIsAutologin;
     bool mIsGoingDown;
     bool mIsProfileLoadingSequence;
 
-    bool mIsClosingDown;
     bool mLF_ON_GA;
-    QString mLine;
-    QMutex mLock;
-    QString mLogin;
-    int mMainIconSize;
-    QString mMudOutputBuffer;
-    int mMXPMode;
     bool mNoAntiAlias;
 
-    QString mPass;
     dlgTriggerEditor* mpEditorDialog;
     QScopedPointer<TMap> mpMap;
     dlgNotepad* mpNotePad;
-    QStringList mParagraphList;
 
-    int mPort;
     bool mPrintCommand;
-    QString mPrompt;
 
     // The following was incorrectly called mRawStreamDump
```

Comment:
This comment really refers to BOTH of the next two members - and explains why and how the original member's functions was spilt.